base/access_provider/importer/model.go

```diff
@@ -0,0 +1,96 @@
+package importer
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/raito-io/cli/base/data_source"
+)
+
+type AccessProviderImport struct {
+	LastCalculated  int64            `yaml:"lastCalculated" json:"lastCalculated"`
+	AccessProviders []AccessProvider `yaml:"accessProviders" json:"accessProviders"`
+}
+
+type AccessProvider struct {
```

Comment:
I couldn't help but notice the similarities between import/export. Wouldn't it be more maintainable to start from a base model that shares the common fields in AccessProvider, Access, ... ? WhatItem is the same, I think, as well as the Action type/enum.